chart/values.yaml

```diff
@@ -912,6 +912,183 @@ hooks:
     # @type: string
     executionTimeout: 30s
 
+csi:
+  # Only supports Vault Enterprise servers.
+  # Toggles the deployment of the Vault Secrets Operator CSI driver. This will deploy the driver and the necessary
+  # resources to the cluster.
+  # @type: boolean
+  enabled: false
+
+  # Host Aliases settings for the vault-secrets-operator-csi pods.
+  # The value is an array of PodSpec HostAlias maps.
+  # ref: https://kubernetes.io/docs/tasks/network/customize-hosts-file-for-pods/
+  # Example:
+  # hostAliases:
+  #   - ip: 192.168.1.100
+  #     hostnames:
+  #     - vault.example.com
+  # @type: array<map>
+  hostAliases: []
+
+  # nodeSelector labels for vault-secrets-operator-csi pod assignment.
+  # @type: map
+  # ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+  # Example:
+  # nodeSelector:
+  #   beta.kubernetes.io/arch: amd64
+  nodeSelector: {}
+  # Toleration Settings for vault-secrets-operator-csi pods.
+  # The value is an array of PodSpec Toleration maps.
+  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+  # @type: array<map>
+  # Example:
+  # tolerations:
+  #  - key: "key1"
+  #    operator: "Equal"
+  #    value: "value1"
+  #    effect: "NoSchedule"
+  tolerations: []
+
+  # Affinity settings for vault-secrets-operator-csi pods.
+  # The value is a map of PodSpec Affinity maps.
+  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+  # Example:
+  # affinity:
+  #   nodeAffinity:
+  #     requiredDuringSchedulingIgnoredDuringExecution:
+  #       nodeSelectorTerms:
+  #       - matchExpressions:
+  #         - key: topology.kubernetes.io/zone
+  #           operator: In
+  #           values:
+  #           - antarctica-east1
+  #           - antarctica-west1
+  affinity: {}
+
+  # Image pull secret to use for private container registry authentication which will be applied to the controllers
+  # service account. Alternatively, the value may be specified as an array of strings.
+  # Example:
+  # ```yaml
+  # imagePullSecrets:
+  #   - name: pull-secret-name-1
+  #   - name: pull-secret-name-2
+  # ```
+  # Refer to https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry.
+  # @type: array<map>
+  imagePullSecrets: []
+
+  # Extra labels to attach to the deployment. This should be formatted as a YAML object (map)
+  extraLabels: {}
+
+  # This value defines additional annotations for the deployment. This should be formatted as a YAML object (map)
+  annotations: {}
+
+  driver:
+    # Image sets the repo and tag of the vault-secrets-operator-csi image to use for the driver.
+    image:
+      pullPolicy: IfNotPresent
+      repository: hashicorp/vault-secrets-operator-csi
+      tag: 0.0.0-dev
```

Comment:
Do we still want this to be the default tag? Or will we change this in a follow-up PR?

Reply:
We'll need to update it when we release.